src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/SwitchRules.groovy

```diff
@@ -72,18 +68,21 @@ class SwitchRules {
         }
     }
 
-    List<FlowEntry> getRulesByCookieType(CookieType cookieType) {
-       northboundService.getSwitchRules(switchId).flowEntries
-                .findAll { new Cookie(it.cookie).getType() == cookieType }
+    List<FlowRuleEntity> getRulesByCookieType(CookieType cookieType) {
+       getRules().findAll { new Cookie(it.cookie).getType() == cookieType }
     }
 
-    List<FlowEntry> getServer42FlowRules() {
-        northboundService.getSwitchRules(switchId).flowEntries
-                .findAll { new Cookie(it.cookie).getType() in [CookieType.SERVER_42_FLOW_RTT_INPUT,
+    List<FlowRuleEntity> getServer42FlowRules() {
+        getRules().findAll { new Cookie(it.cookie).getType() in [CookieType.SERVER_42_FLOW_RTT_INPUT,
                                                                CookieType.SERVER_42_FLOW_RTT_INGRESS] }
     }
 
-    List<FlowEntry> getRules() {
-        northboundService.getSwitchRules(switchId).flowEntries
+    List<FlowRuleEntity> getServer42ISLRules() {
+        getRules().findAll { (new Cookie(it.cookie).getType() in [CookieType.SERVER_42_ISL_RTT_INPUT] ||
+                it.cookie in [Cookie.SERVER_42_ISL_RTT_TURNING_COOKIE, Cookie.SERVER_42_ISL_RTT_OUTPUT_COOKIE]) }
+    }
+
+    List<FlowRuleEntity> getRules() {
+        northboundService.getSwitchRules(switchId).flowEntries.collect { new FlowRuleEntity(it) }
```

Comment:
From developer point of view this is quite unusual approach to have service level logic(northbound service, database) in the model classes.
I noticed that this is not the only one place in our func-test code with the similar approach, and we have similar logic in other model classes in func test packages.

Do not treat it as a call to action:)